Makefile

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2024 Toitware ApS.
+# Use of this source code is governed by a Zero-Clause BSD license that can
+# be found in the LICENSE file.
+
+# Change to your configuration. See ../toit/toolchains for the available targets.
```

Comment:
Is `../toit/` the right path? It looks like it should just be `toit/`.

Reply:
good catch.
done.